exporter/collector/metrics.go

```diff
@@ -260,7 +260,7 @@ func (me *MetricsExporter) PushMetrics(ctx context.Context, m pmetric.Metrics) e
 				recordPointCountDataPoint(ctx, failedPoints, st)
 			}
 			if err != nil {
-				errs = append(errs, fmt.Errorf("failed to export time series to GCM: %v", err))
+				errs = append(errs, fmt.Errorf("failed to export time series to GCM: %+v", s))
```

Comment:
For a further refinement: should we include the original error as a cause of what we're generating?

```go
errs = append(errs, fmt.Errorf("failed to export time series to GCM (status %+v): %w, s, err)
```
```